ddtrace/appsec/processor.py

```diff
@@ -229,6 +230,10 @@ def on_span_finish(self, span):
                 return
             if _Addresses.SERVER_REQUEST_HEADERS_NO_COOKIES in data:
                 _set_headers(span, data[_Addresses.SERVER_REQUEST_HEADERS_NO_COOKIES])
+
+            if _Addresses.SERVER_REQUEST_COOKIES in data:
```

Comment:
We don't need to collect the cookies, if an attack happens on them, the WAF output will have them